perfkitbenchmarker/providers/aws/aws_virtual_machine.py

```diff
@@ -545,7 +545,10 @@ class RhelBasedAwsVirtualMachine(AwsVirtualMachine,
 
   def __init__(self, vm_spec):
     super(RhelBasedAwsVirtualMachine, self).__init__(vm_spec)
-    self.user_name = 'ec2-user'
+    if FLAGS.aws_user_name:
```

Comment:
Hrm, yes you are right.  Is there a way to see if the user passed in a value or if it is using the default one?  This is trickier than I thought it was going to be.

I could also make a CentOSBasedAwsVirtualMachine that sets its user name to centos, but that sounds a little messy.